qiskit_experiments/library/characterization/t1_analysis.py

```diff
@@ -31,26 +31,30 @@
 class T1Analysis(BaseAnalysis):
     r"""A class to analyze T1 experiments.
 
-    Fit Model
+    #section: fit Model
         The fit is based on the following decay function.
 
         .. math::
 
             F(x) = a e^{-x/t1} + b
 
-    Fit Parameters
-        - :math:`amplitude`: Height of the decay curve
-        - :math:`offset`: Base line of the decay curve
-        - :math:`t1`: This is the fit parameter of main interest
+    #section: fit Parameters
+       defpar a:
+           desc: Height of the decay curve.
+           init_guess: Determined by :math:`(y_0 - b)`.
 
-    Initial Guesses
-        - :math:`amplitude\_guess`: Determined by :math:`(y_0 - offset\_guess)`
-        - :math:`offset\_guess`: Determined by the last :math:`y`
-        - :math:`t1\_guess`: Determined by the mean of the data points
+       defpar b:
+           desc: Base line of the decay curve.
+           init_guess: Determined by the last :math:`y`.
+
+       defpar t1:
+           desc: This is the fit parameter of main interest.
+           init_guess: Determined by the mean of the data points.
     """
 
     @classmethod
     def _default_options(cls):
+        """Default analysis options"""
```

Comment:
You need to list descriptions introduced in this subclass here. i.e.
```suggestion
        """Default analysis options
        
        Analysis Options:
            t1_guess (Type): Doc
            amplitude_guess (Type): Doc
            offset_guess (Type): Doc
        """
```